README.md

```diff
@@ -2,14 +2,17 @@
   <img width="100%" src="https://assets.solidjs.com/banner?type=CLI&background=tiles&project=%20" alt="Solid CLI">
 </p>
 
-# Solid CLI (This is currently very much still in beta)
+> [!WARNING]
+> 🚧 This project is under heavy development and still in beta. More features coming soon!
+
+# Solid CLI
 
 [![Build and Test](https://github.com/solidjs-community/solid-cli/actions/workflows/tests.yml/badge.svg)](https://github.com/solidjs-community/solid-cli/actions/workflows/tests.yml)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/5233ac74-3f53-4c90-b95d-25b528b931a1/deploy-status)](https://app.netlify.com/sites/solid-cli/deploys)
 
-A custom CLI built for the purpose of installing and managing SolidJS apps and projects. The goal of the CLI is to provide a useful and powerful utility to installing any dependencies, searching the Solid ecosystem etc.
+A custom-built CLI for creating and managing SolidJS apps and projects.
```

Comment:
I shortened the description a bit here but I couldn't think of anything else to add maybe list... features? more details?